lib/isodatetime/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-#-----------------------------------------------------------------------------
+# ----------------------------------------------------------------------------
 # (C) British Crown Copyright 2013-2014 Met Office.
```

Comment:
Actually, I have forgotten to do this for this isodatetime release. I'll get that done tomorrow.